Sources/CgRPC/shim/cgrpc.h

```diff
@@ -126,17 +151,22 @@ void grpc_shutdown(void);
 const char *grpc_version_string(void);
 const char *grpc_g_stands_for(void);
 
+char* gpr_strdup(const char* src);
+
 void cgrpc_completion_queue_drain(cgrpc_completion_queue *cq);
 void grpc_completion_queue_destroy(cgrpc_completion_queue *cq);
 
 // helper
 void cgrpc_free_copied_string(char *string);
 
 // channel support
-cgrpc_channel *cgrpc_channel_create(const char *address);
+cgrpc_channel *cgrpc_channel_create(const char *address, 
+                                    grpc_arg *args,
```

Comment:
Can this be made `const`? (Might not be the case, but worth checking.)

Reply:
No it can't: 
```
typedef struct {
  size_t num_args;
  grpc_arg *args;
} grpc_channel_args;
```

Reply:
I see, thanks! Was worth a check ;-)